integrations/tensorflow/iree_tf_compiler/TFL/LowerGlobalTensors.cpp

```diff
@@ -131,8 +150,19 @@ class LowerGlobalTensorsPass
       if (!address) continue;
 
       builder.setInsertionPoint(assign);
+      Value value = assign.value();
+      Type storageType = address.getType()
+                             .cast<iree_compiler::IREE::Util::PtrType>()
+                             .getTargetType();
+      if (storageType != value.getType()) {
+        value = builder
+                    .create<UnrealizedConversionCastOp>(assign.getLoc(),
+                                                        storageType, value)
+                    .getResult(0);
```

Comment:
What's going to make sure this goes away? Do we have something like `-reconcile-unrealized-conversion-casts` after this pass or something? Just want to make sure it doesn't bleed through too far?

Reply:
So in theory it should be cleaned up by the remove quantized type pass and canonicalizer. That problem is that IREE opts don't support quantized types so I have to wait until they are stripped.

Include the reconcile-unrealized-conversion-casts pass in case that is not sufficient.

Reply:
The other nice thing about that pass is that it gives you an error if there are any casts still left.